lms/static/js/spec/search/search_spec.js

```diff
@@ -89,6 +89,7 @@ define([
 
         beforeEach(function () {
             TemplateHelpers.installTemplate('templates/courseware_search/search_item');
+            TemplateHelpers.installTemplate('templates/courseware_search/search_item_seq');
```

Comment:
So it's not very different. One the face of it this is a fine change, but I am afraid of setting a precedent of a new template file for each type of search result, it's possible that we start to have many more types - but I suppose we can cross that bridge when we come to it. Thoughts?